src/python/pants/engine/legacy/graph.py

```diff
@@ -381,9 +382,13 @@ def hydrate_bundles(bundles_field, snapshot_list):
   for bundle, filespecs, snapshot in zipped:
     spec_path = bundles_field.address.spec_path
     kwargs = bundle.kwargs()
+    # NB: We `include_dirs=True` because bundle filesets frequently specify directories in order
+    # a (deprecated) default inclusion of their recursive contents. See the related deprecation
+    # in `pants.backend.jvm.tasks.bundle_create`.
```

Comment:
`...frequently specify directories in order a (deprecated) default inclusion of their...` eh?